ql/src/java/org/apache/hadoop/hive/ql/plan/ImportTableDesc.java

```diff
@@ -175,7 +176,7 @@ public TableType tableType() {
     return TableType.MANAGED_TABLE;
   }
 
-  public Table toTable(HiveConf conf) throws Exception {
+  public Table toTable(HiveConf conf) throws HiveException {
```

Comment:
Will the CreateTableLike operation also use the same code path? It would be good to a test case for the same.

Reply:
I think we don't need to, this method will only throw HiveException.